components/related-donors-table.js

```diff
@@ -0,0 +1,41 @@
+// node_modules
+import Link from "next/link";
+import PropTypes from "prop-types";
+// components
+import { DataGridContainer } from "./data-grid";
+import SortableGrid from "./sortable-grid";
+
+const relatedDonorsColumns = [
+  {
+    id: "related_donor_id",
+    title: "Donor",
+    display: ({ source }) => {
+      return <Link href={source.donor["@id"]}>{source.donor.accession}</Link>;
+    },
+  },
+  {
+    id: "relationship_type",
+    title: "Relationship Type",
+    display: ({ source }) => source.relationship_type,
```

Comment:
I doubt you need the `display` property at all. If you don’t have one, it just gets the property matching `id` and displays it, which is all this `display` function does. You need `display` for `related_donor_id` in the first column because the built-in mechanism only displays contents and can’t make a link out of it, but the built-in mechanism can totally handle this one.